### DISCUSSION TITLE
spherical light

### DISCUSSION THREAD

**Discussion by Jureq1234** [pyvista/pyvista#1773] [Q&A]
Hi,

It is possible to create spherical light using pv.Light? Or maybe there is another light type that is available to create spherical light?
Regards,
Jurek

**Comment by tkoyama010** [comment]
Hi @Jureq1234 .
Is [Light.positional](https://dev.pyvista.org/api/core/_autosummary/pyvista.Light.positional.html) is what you want?

**Reply by Jureq1234** [reply]
Yes, but as I have checked, positional light does not drop any shadows, which I really need.
![lpos](https://user-images.githubusercontent.com/88145971/140026040-1e34c8aa-e290-4edb-9e27-6f3d8f7c2f4a.JPG)
